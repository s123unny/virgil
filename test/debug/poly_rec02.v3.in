b ../core/poly_rec02.v3 11
run
bt
info l
c
q
