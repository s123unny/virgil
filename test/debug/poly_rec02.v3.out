breakpoint|../core/poly_rec02.v3 11|0
stopOnBreakpoint
bt|poly_rec02.main|../core/poly_rec02.v3|11
variable|0|this|poly_rec02|poly_rec02|false
variable|1|arg|0|int|false
variable|2|o|null|poly_rec02a<poly_rec02a<int>>|false
end
result|0
